tests/plugins/autocomplete/manual/__template__.html

```diff
@@ -30,6 +26,10 @@ <h1>Lorem ipsum dolor sit amet, consectetur.</h1>
 		}
 	} );
 
+	// The body of the root html element has additional margin-left to display bender manual-sidebar which has fixed position.
+	// Our autcomplete view lives inside root html so with its margin it has invalid position relative to the editor.
+	// We have to change position of our editor relative to the root html by setting position:static ( see style tag ) and move it to the center of the page
+	// so it won't be hidden below sidebar.
```

Comment:
Still this workaround is way over-complicated. Am I correct to say that simply setting:
```html
<style>
	body {
		margin-left: 0px;
		padding-left: 350px;
	}
</style>
```

Solves the problem? Make sure to still leave a short comment for the reason of the workaround.